.changeset/eight-pillows-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```

Comment:
Since the public API of this component is changing (specifically `buttonType` is being broadened to include a new union member) could we mark this as `minor`?

Reply:
Definitely! Switched it to `minor`